index.html

```diff
@@ -1,14 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>My Website</title>
+    <meta name="description" content="Kingnoel Portfolio">
+    <meta name="keywords" content="HTML, CSS, JavaScript">
```

Comment:
- [ ] Hey @ridabensalem, you have added some of the requirements which is a great and awesome job, but please take a look at this image below to understand what is missing, you should add a link to each change you have brought to the work, in case you did not found any issue to adjust, the requirement says that you should point it to no issue found in the pull request.

<img width="1114" alt="1" src="https://user-images.githubusercontent.com/77843050/208705160-32e9d998-837b-4964-a712-a9c003edb182.png">